src/AppBundle/Resources/config/services.yml

```diff
@@ -1,101 +1,87 @@
-parameters:
-    app.installer.class: AppBundle\Installer\AppInstaller
-    app.criteria.menu.class: AppBundle\Criteria\Menu
-    app.controller.menu.class: AppBundle\Controller\MenuController
-    app.criteria.children.class: AppBundle\Criteria\Children
-    app.controller.blog.class: AppBundle\Controller\BlogController
-    app.controller.home.class: AppBundle\Controller\HomeController
-    app.controller.contactform.class: AppBundle\Controller\ContactFormController
-    app.form.type.contact.class: AppBundle\Form\Type\ContactType
-    app.form.contact.form.class: AppBundle\Form\Contact\Form
-    app.entity.message.class: AppBundle\Entity\Message
-    app.mail.sender.class: AppBundle\Mail\Sender
-    app.controller.gallery.class: AppBundle\Controller\GalleryController
```

Comment:
I would really consider this change. Maintaining those is really extra work, imho.